examples/camera/app.js

```diff
@@ -71,16 +72,15 @@ export default function App() {
     });
   }, [viewStateA, viewStateB]);
 
-  const getDeckScene = timeline => {
```

Comment:
Previously users had to provide a callback so that deck could supply the timeline. It's confusing and not necessary.